deploy/deploy.sh

```diff
@@ -41,7 +42,7 @@ if [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
   export TARGET_ENV="project-template-$target_env"
 
   # This needs to be exported because our docker-compose.yml below is interpolating it
-  export CONTAINER_REPO=680542703984.dkr.ecr.us-east-1.amazonaws.com/project-template
+  export CONTAINER_REPO=ECR_ENDPOINT
```

Comment:
your update on line 30 above actually negates the need for this to even exist. the environment prefixed env var that is set in travis `environment_ECR_ENDPOINT` will be exported on line 30 above.

Reply:
Ok, I removed `CONTAINER_REPO` and referenced it directly as `$ECR_ENDPOINT` in the evaluated strings below.